src/Utils/systems.jl

```diff
@@ -14,8 +14,11 @@ const CACCS = ConstrainedAffineControlContinuousSystem
 const CACDS = ConstrainedAffineControlDiscreteSystem
 const CACS = ConstrainedAffineContinuousSystem
 const CADS = ConstrainedAffineDiscreteSystem
+const BBCS = Union{BlackBoxContinuousSystem,
+                   ConstrainedBlackBoxContinuousSystem,
+                   ConstrainedBlackBoxControlContinuousSystem}
 
-export LCS, LDS, CLCS, CLDS, CLCCS, CLCDS, CACCS, CACDS, IVP
+export LCS, LDS, CLCS, CLDS, CLCCS, CLCDS, CACCS, CACDS, IVP, BBCS
```

Comment:
I think exporting a `Union` is not helpful because it cannot be used for constructing a system.